app/routes/questionnaire.py

```diff
@@ -109,9 +109,8 @@ def get_questionnaire(schema, questionnaire_store):
         list_store=questionnaire_store.list_store,
     )
 
-    are_hub_required_sections_complete = all(
-        questionnaire_store.progress_store.is_section_complete(section_id)
-        for section_id in schema.get_section_ids_required_for_hub()
+    are_hub_required_sections_complete = get_hub_required_sections(
```

Comment:
I think we need to change this because of the method `get_hub_required_sections` is doing more than what it says.

Maybe the method name should be `are_hub_required_sections_complete` and var can be something like `hub_required_sections_complete`?

Thoughts?